front/app/components/FormBuilder/edit/index.tsx

```diff
@@ -93,10 +95,7 @@ const FormEdit = ({
     phaseId: isFormPhaseSpecific ? phaseId : undefined,
   });
 
-  const { data: customForm } = useCustomForm({
-    projectId,
-    phaseId: isFormPhaseSpecific ? phaseId : undefined,
-  });
```

Comment:
It's too error-prone to hope devs will think of checking isFormPhaseSpecific each time they use useCustomForm (or the fields variant).